src/test/integration/IntegrationBase.t.sol

```diff
@@ -2307,6 +2324,21 @@ abstract contract IntegrationBase is IntegrationDeployer, TypeImporter {
         return expectedShares;
     }
 
+    /// @dev For some strategies/underlying token balances, calculate the expected shares received
+    /// Exact same as _calculateExpectedShares but for multiple stakers
+    function _calculateExpectedShares(
+        IStrategy[][] memory strategies,
+        uint[][] memory tokenBalances
```

Comment:
The arrayification of integration tests is getting unwieldy. It might be time to start introducing some helper structs.